runner.go

```diff
@@ -137,10 +137,15 @@ func Execute(c *Context, job *SubmittedJob) {
 	job.StartedAt = StoreTime(time.Now())
 	job.QueueDelay = job.StartedAt.AsTime().Sub(job.CreatedAt.AsTime()).Nanoseconds()
 
+	image := c.DefaultImage
+	if len(job.Layers) != 0 {
+		image = job.Layers[0].Name
```

Comment:
:+1: That's cool.

It might be better to have an explicit error if `len(job.Layers) > 1` as well, rather than silently ignoring them.